src/es/components/organisms/stage/Stage.js

```diff
@@ -0,0 +1,91 @@
+// @ts-check
+import { Shadow } from '../../web-components-toolbox/src/es/components/prototypes/Shadow.js'
+
+/**
+ * Stage
+ * An example at: src/es/components/pages/Home.html
+ *
+ * @export
+ * @class EventList
+ * @type {CustomElementConstructor}
+ */
+
+export default class Stage extends Shadow() {
+  constructor (options = {}, ...args) {
+    super({ importMetaUrl: import.meta.url, ...options }, ...args)
+  }
+
+  connectedCallback () {
+    if (this.shouldRenderCSS()) this.renderCSS()
+  }
+
+  /**
+   * evaluates if a render is necessary
+   *
+   * @return {boolean}
+   */
+  shouldRenderCSS () {
+    return !this.root.querySelector(`:host > style[_css], ${this.tagName} > style[_css]`)
+  }
+
+  renderCSS () {
+    this.css = /* css */`
+      :host {
+        position: relative;
+      }
+
+      :host .stage__overlay {
+        background-color: var(--steps-color-black);
+        opacity: 0.5;
+        z-index: 20;
+        position: absolute;
+        height: 100vh;
+        width: 100vw;
+        top: 0;
+        left: 0;
+        right: 0;
+        bottom: 0;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+      }
+
+      :host .stage__content {
+        z-index: 30;
+        display: flex;
+        flex-direction: column;
+      }
+
+      :host .stage__content h2 {
+        font-size: 4.8rem;
+        font-weight: 400;
+        color: var(--steps-color-white);
+        text-align: center;
+        margin: 0;
+      }
+
+      :host .stage__content span {
+        font-size: 4.8rem;
```

Comment:
Because of the unwanted font reset side effects, we should take the given font sizes here

```css
font-size: var(--h1-font-size);
```